TurboStack App/howto_newuser.md

```diff
@@ -51,3 +49,28 @@ system_users:
 Now an account is created. Applications can be installed.
 
 Example, [Deploy a Magento2 application](./howto_newapp.md)
+
+## Cloning an account
+
+To set up a new account with the same configuration as an existing one, you can easily use the `Clone Account` feature. This function replicates the entire configuration, enabling a quick and efficient setup with minimal effort.
```

Comment:
To set a clone a complete copy (files and database) of an existing application ...